src/elements/funnel-progress/src/stories.tsx

```diff
@@ -57,7 +57,7 @@ export const AutomatedTests = () => {
   })
 
   return (
-    <AllThemes themes={['uswitch', 'money', 'journey']}>
+    <AllThemes themes={['uswitch']}>
```

Comment:
Do the changes need to be made to the Journey theme too, or we holding off releasing this until the theme is switched over?

Reply:
good question, i don't know